client/src/utils/chooserModal.js

```diff
@@ -0,0 +1,143 @@
+import $ from 'jquery';
+import { gettext } from './gettext';
+
+const submitCreationForm = (modal, form, errorContainerSelector) => {
```

Comment:
similar here, I think the first two (DOM elements) can stay as is but the last one should become an options object.

```suggestion
const submitCreationForm = (modal, form, { errorContainerSelector } = {}) => {
```

Just makes it easier to add more things later and also means it has to be called with a clearer 'name' for these thigns.

e.g. `submitCreationForm(myModal, myForm { errorContainerSelector: '.error-container' })`